sklearn/datasets/_samples_generator.py

```diff
@@ -132,6 +133,7 @@ def make_classification(n_samples=100, n_features=20, *, n_informative=2,
         for reproducible output across multiple function calls.
         See :term:`Glossary <random_state>`.
 
+
```

Comment:
nit: do we really want 2 blank lines, here? It seems unusual.

```suggestion
```

Reply:
Probably a miss-click. I must have missed this because PEP8 check passed. I'll correct this. Thanks!